sound/soc/sof/sof-pci-dev.c

```diff
@@ -283,9 +283,10 @@ static int sof_pci_probe(struct pci_dev *pci,
 
 	ret = snd_intel_dsp_driver_probe(pci);
 	if (ret != SND_INTEL_DSP_DRIVER_ANY &&
-	    ret != SND_INTEL_DSP_DRIVER_SOF)
+	    ret != SND_INTEL_DSP_DRIVER_SOF) {
+		dev_dbg(&pci->dev, "SOF PCI driver not selected, aborting probe\n");
```

Comment:
looks ok but why only dbg if we're returning an error?

Reply:
it's not really an error, it means that we let another driver probe.
In practice, you will see the hdaudio legacy driver try and probe first. If the rsults of snd_intel_dsp_driver_probe() indicate that e.g. DMIC or SoundWire are present, or for a GLK+ chromebook, it will bail, and then the SOF driver will probe.